src/main/java/xyz/nucleoid/plasmid/game/stats/GameStatisticBundle.java

```diff
@@ -0,0 +1,81 @@
+package xyz.nucleoid.plasmid.game.stats;
+
+import com.google.gson.JsonObject;
+import it.unimi.dsi.fastutil.objects.Object2ObjectMap;
+import it.unimi.dsi.fastutil.objects.Object2ObjectOpenHashMap;
+import net.minecraft.server.network.ServerPlayerEntity;
+import xyz.nucleoid.plasmid.util.PlayerRef;
+
+import java.util.Map;
+import java.util.UUID;
+
+/**
+ * Wrapper containing a map players' {@link StatisticBundle}s
+ */
+public class GameStatisticBundle {
+    private final Object2ObjectMap<UUID, StatisticBundle> players = new Object2ObjectOpenHashMap<>();
```

Comment:
Could be interesting to also provide a method for storing global statistics- things such as total games played / average win rates for certain teams etc. If games can retrieve statistics for themselves, they could potentially do interesting things with that data- or it could be used for balancing.

Reply:
This would also require changes to the backend for storage too, but I can add an additional field to `GameStatisticBundle` for global statistics so that an API can be exposed in plasmid